readme.md

```diff
@@ -425,7 +412,7 @@ would match:
 Assert that there is one element, and that it passes the expectations of a function.
 
 ```js
-var promise = scope.shouldHaveElement(fn, [options]);
+const promise = scope.shouldHaveElement(fn, [options]);
```

Comment:
Maybe we should change these to just have await:

```js
await scope.shouldHaveElement(fn, [options]);
```
as that is what we do in "real life"

Reply:
Trying this "insert a suggestion" button. What is it going to do?

```suggestion
await scope.shouldHaveElement(fn, [options]);
```